cli/aws_ddk/__main__.py

```diff
@@ -314,6 +319,7 @@ def create_repository(
     required=False,
 )
 def deploy(
+    stacks: Iterable[str] = ["--all"],
```

Comment:
Is it possible to avoid mutable default arguments? One of common python's [gotchas](https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments) that may lead to unpredictable behavior